### ISSUE TITLE
notice: An error occured fetching the exchange data .. not all skeletons will be available to you undefined {}`

### ISSUE THREAD

**Issue by guaka** [docpad/docpad#930]
*[notgivingup](https://twitter.com/guaka/status/567455944196759553)*

`notice: An error occured fetching the exchange data, this is okay, but not all skeletons will be available to you undefined {}`

```
$ docpad run
info: Welcome to DocPad v6.71.1 (global installation: /usr/local/lib/node_modules/docpad)
notice: If everyone who saw this message donated $1/week, maintaining DocPad would become sustainable: http://docpad.org/donate
info: Contribute: http://docpad.org/docs/contribute
info: Plugins:
info: Environment: development
info: Updating the exchange...
This can take a moment...
notice: An error occured fetching the exchange data, this is okay, but not all skeletons will be available to you undefined {}
info: You are about to create your new project inside your current directory. Below is a list of skeletons to bootstrap your new project:

  1.    No Skeleton
        Prefer to start from scratch? You can get started without any skeleton if you wish

Which skeleton will you use? [1-1]
  1.    No Skeleton
```

I don't have any proxy set up? Any way to get some more verbosity?

**Comment by balupton** [comment]
Use the `-d` or `--debug` flag, so `docpad run -d`

**Comment by balupton** [comment]
That's really weird though... That should not happen.

**Comment by guaka** [comment]
Actually, I tried `-d` but and it is more verbose, just not in a very useful way (at least to me).
```
info: Contribute: http://docpad.org/docs/contribute
    → [2015-02-19 17:26:07.398] [/usr/local/lib/node_modules/docpad/out/lib/docpad.js:1242] [DocPad.ready]
info: Plugins:
    → [2015-02-19 17:26:07.400] [/usr/local/lib/node_modules/docpad/out/lib/docpad.js:1243] [DocPad.ready]
info: Environment: development
    → [2015-02-19 17:26:07.405] [/usr/local/lib/node_modules/docpad/out/lib/docpad.js:1244] [DocPad.ready]
debug: Emitting the event: welcome
    → [2015-02-19 17:26:07.420] [/usr/local/lib/node_modules/docpad/out/lib/docpad.js:1163] [DocPad.emitSerial]
debug: Emitted the event: welcome
    → [2015-02-19 17:26:07.428] [/usr/local/lib/node_modules/docpad/out/lib/docpad.js:1168] [TaskGroup.<anonymous>]
debug: Emitting the event: docpadReady
    → [2015-02-19 17:26:07.434] [/usr/local/lib/node_modules/docpad/out/lib/docpad.js:1163] [DocPad.emitSerial]
debug: Emitted the event: docpadReady
    → [2015-02-19 17:26:07.437] [/usr/local/lib/node_modules/docpad/out/lib/docpad.js:1168] [TaskGroup.<anonymous>]
info: Updating the exchange...
This can take a moment...
    → [2015-02-19 17:26:07.446] [/usr/local/lib/node_modules/docpad/out/lib/docpad.js:2693] [DocPad.getExchange]
debug: Loading configuration url: http://docpad.org/exchange.json?version=6.71.1
    → [2015-02-19 17:26:07.450] [/usr/local/lib/node_modules/docpad/out/lib/docpad.js:1563] [DocPad.loadConfigUrl]
notice: An error occured fetching the exchange data, this is okay, but not all skeletons will be available to you undefined {}
    → [2015-02-19 17:26:09.110] [/usr/local/lib/node_modules/docpad/out/lib/docpad.js:2698] [unknown]
info: You are about to create your new project inside your current directory. Below is a list of skeletons to bootstrap your new project:
```

**Comment by guaka** [comment]
Or it is... `wget http://docpad.org/exchange.json?version=6.71.1` works for me but `curl http://docpad.org/exchange.json?version=6.71.1` doesn't!

**Comment by guaka** [comment]
Could that be something? Looks like my `curl` doesn't like 301s. `curl https://raw.githubusercontent.com/bevry/docpad-extras/docpad-6.x/exchange.json` works fine.

**Comment by balupton** [comment]
Looking into it.

**Comment by balupton** [comment]
Fixed in DocPad v6.72.0. Cheers!

**Comment by guaka** [comment]
Thanks!
